dask_expr/_expr.py

```diff
@@ -1595,9 +1595,20 @@ def _simplify_up(self, parent, dependents):
     def _lower(self):
         if not isinstance(self, BlockwiseHead):
             # Lower to Blockwise
+            if hasattr(self.frame, "_partitions") and self.npartitions > len(
```

Comment:
This goes back to the class PartitionsFiltered, so you can do an isinstance check

Reply:
https://github.com/dask-contrib/dask-expr/pull/636/commits/63d15469bf460e969021f55f001c10e3b657f9d5, thanks!